### DISCUSSION TITLE
Manually set a default search engine

### DISCUSSION THREAD

**Discussion by infoekcz** [bromite/bromite#2617] [Q&A]
Hello,
why Bromite doesn't allow to manually set a default search engine that is not in the preset list? I would like to set up a Xoogle.cz search engine.
Xoogle for Czechia: https://xoogle.cz/search.php?q=%s
Xoogle for Slovakia: https://sk.xoogle.cz/search.php?q=%s

Thank you for the answers!

**Comment by DKB-91** [comment]
I don't think it is possible for any Chromium browser to add a custom search engine, only Firefox and it's derivatives (Fennec F-Droid, Mull) can.

You can add one to Bromite and use it by doing a search and then going to Search Engines and selecting it (if it appears in recently visited), but I don't think there is a way to add both of them permanently in the list. (Only in Firefox Android based browsers a user can edit their search engines list)

Also to note you will lose it and have to repeat the process if you change to another search engine.
And I would not use Bromite either as it is outdated and lack security patches, change web browser or to use a more up to date beta version search for uazo's bromite-buildtools repository. (Unless you're on a 32 bit device, then switch to another browser).
